integreat_cms/static/src/js/filter-form.ts

```diff
@@ -14,6 +14,9 @@ const resetToDefaultValue = (node: HTMLSelectElement | HTMLInputElement) => {
         // Non-checkbox input marked to have a default value
         /* eslint-disable-next-line no-param-reassign */
         node.value = node.getAttribute("data-default-value");
+    } else if (node.matches("select")) {
+        const idName = node.id;
```

Comment:
```suggestion
        const inputElement = document.getElementById(node.id) as HTMLInputElement
        inputElement.value = ""
```
I'd prefer to write it like this, as I feel this is better use of the variable and more legible. But I suppose this comes down to personal preference, so feel free to disagree and not follow the suggestion